packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -527,6 +527,45 @@ class AnimationController extends Animation<double>
     return _animateToInternal(lowerBound);
   }
 
+  /// Toggles the direction of this animation, based on whether it's [aimedForward].
+  ///
+  /// Specifically, this function acts the same way as [reverse] if the [status] is
+  /// either [AnimationStatus.forward] or [AnimationStatus.completed], and acts as
+  /// [forward] for [AnimationStatus.reverse] or [AnimationStatus.dismissed].
+  ///
+  /// The most recently returned [TickerFuture], if any, is marked as having been
+  /// canceled, meaning the future never completes and its [TickerFuture.orCancel]
+  /// derivative future completes with a [TickerCanceled] error.
+  TickerFuture toggle({ double? from }) {
```

Comment:
Can you document the parameter?

Reply:
Sure can!

I copied the documentation over from the `forward`/`reverse` functions, so I'll document the parameter for those as well.